frappe/core/doctype/user/user.py

```diff
@@ -279,7 +283,7 @@ def send_login_mail(self, subject, template, add_args, now=None):
 		from frappe.utils import get_url
 
 		mail_titles = frappe.get_hooks().get("login_mail_title", [])
-		title = frappe.db.get_default('company') or (mail_titles and mail_titles[0]) or ""
+		title = (mail_titles and mail_titles[0]) or frappe.db.get_default('company') or ""
```

Comment:
can you move this via hook in erpnext too, so we clean it up completely?

Reply:
Will do it. @rmehta I also believe the hook must be renamed to welcome_email_subject , what do you suggest ?

Reply:
👍 

A better solution would be to set a default `site_name` property in System Settings (set default to "My Site"), and set company name to `site_name` in erpnext setup. This way we don't need to write a hook for in the default frappe install.

Better UX !